src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -49,17 +53,62 @@ public class Bundler
             RuntimeConfigJson = appAssemblyName + ".runtimeconfig.json";
             RuntimeConfigDevJson = appAssemblyName + ".runtimeconfig.dev.json";
 
-            BundleManifest = new Manifest(Target.BundleVersion, netcoreapp3CompatMode: options.HasFlag(BundleOptions.BundleAllContent));
+            BundleManifest = new Manifest(Target.BundleMajorVersion, netcoreapp3CompatMode: options.HasFlag(BundleOptions.BundleAllContent));
             Options = Target.DefaultOptions | options;
         }
 
+        private bool ShouldCompress(FileType type)
+        {
+            // compression is not supported before bundle vesion 3
+            if (Target.BundleMajorVersion < 3)
```

Comment:
Personally I would prefer if we changed the bundle version to be the .NET major version - otherwise it's a versioning mess. Can we change the major version of the bundle for 6 to 6?

Reply:
`6` would work too. Might be easier to remember.

Reply:
The only downside is that there will be holes in the versions. For example if in .NET 7 we don't need a new version but in .NET 8 we will, there will be `6` and `8`, but no `7`. I don't think it matters - and as you said, it will be easy to map to released.